it/src/main/scala/CleanupTestEvents.scala

```diff
@@ -0,0 +1,46 @@
+
+
+
+
+import scala.concurrent.Await
+import scala.concurrent.duration.DurationInt
+import scala.util.Try
+import org.slf4j.LoggerFactory
+import org.zalando.nakadi.client.scala.model.EventType
+import org.zalando.nakadi.client.scala.Client
+import org.zalando.nakadi.client.scala.ClientFactory
+
+object CleanupCreatedTestEventTypes extends App {
+//  private val log = LoggerFactory.getLogger(ClientFactory.getClass)
+
+  deleteAllEventsCreatedByClientIntegrationTests()
+
+  def deleteAllEventsCreatedByClientIntegrationTests() = {
+    val eventNames = List("SimpleEvent", "ClientIntegrationTest", "PaymentCreatedDataChangeEventTest")
+    val client = ClientFactory.buildScalaClient()
+    val eventsOnNakadi = Await.result(client.getEventTypes(), 25.second)
+    val filteredEvents = eventsOnNakadi match {
+      case Right(Some(events)) if !events.isEmpty=>
+        val result = events.filter(p=> eventNames.exists(s =>p.name.startsWith(s)) && shouldBeRemoved(p.owningApplication) ||shouldBeRemoved(p.owningApplication) )
+        println("######################### Number of filtered events to be deleted: "+result.size)
```

Comment:
This is a "command-line" tool and does not work with logging and was specially made for this a single purpose: Delete created events.